react/ShelfContent.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import Slider from 'react-slick'
+import Arrow from './Arrow'
+import Dots from './Dots'
+import ShelfItem from './ShelfItem'
+
+import VTEXClasses from './CustomClasses'
+import ScrollTypes from './ScrollTypes'
+
+/**
+ * ShelfContent Component. Realizes the interaction with react-slick
+ * and render the properly content of the Shelf depending of edit mode state.
+ */
+class ShelfContent extends Component {
+  BREAKPOINT_MEDIUM_VIEWPORT = 1024
```

Comment:
You could extract those constants to outside the class and avoid to call as this.BREAKPOINT...